instances/technique/supportVectorMachineClassifier.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/supportVectorMachineClassifier",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "A 'support-vector machine classifier' is a supervised machine learning technique that analyze data for classification and regression analysis.",
```

Comment:
... technique that analyze**s** data ...

Reply:
done